docs/vereisten/index.md

```diff
@@ -1,69 +1,88 @@
-    Meer vereisten volgen in een volgende versie van het Algoritmekader. 
-
-## Welke vereisten zijn wanneer van toepassing?
-In dit deel van het Algoritmekader kan worden genavigeerd door de verschillende vereisten. 
-Wat precies van toepassing is, is sterk afhankelijk van de specifieke toepassing. 
-Daarom worden de vereisten gekoppeld aan het type technologie (rekenregel, machine learning of generatieve AI) en aan de risico classificatie (niet-impactvol, impactvol en hoge risico). 
-Dit moet gebruikers helpen om te duiden welke vereisten in welke situaties moeten worden nageleefd. 
-
-??? note "Opmerking"
-    Aan deze functionaliteit wordt nog gewerkt. Meer hierover zal volgen in een volgende versie. 
+## Aantal eisen verschilt per situatie
+Welke eisen gelden voor jouw organisatie, hangt af van:
+- de technologie die je gebruikt: rekenregels, machinelearning of generatieve AI
+- het risiconiveau van je product
+- de [fase in de levenscyclus](https://minbzk.github.io/Algoritmekader/levenscyclus/) van je project
 
 ### Voorbeeld
-Ter illustratie, ongeacht het type technologie, zijn discriminerende algoritmes en AI verboden en het vereiste van non-discriminatie moet in alle gevallen worden toegepast. 
-De mate waarin dit speelt kan verschillen en het aantal te treffen maatregelen ook. 
-Organisaties zullen hier hoe dan ook aandacht aan moeten besteden. 
-Dit is anders als het gaat om bijvoorbeeld het vereiste om algoritmes en AI op te nemen in het Algoritmeregister. 
-Dit is van belang bij impactvolle algoritmische toepassingen en hoog-risico AI-systemen. 
-Voor de niet-impactvolle rekenregels is dit niet noodzakelijk en kan daardoor achterwege blijven. 
-
-## Overzicht van vereisten
-
-Onderstaand volgt een overzicht van vereisten per fase van de levenscyclus. 
-
-??? simple "Probleemanalyse"
-
-    <!-- list_vereisten_1 levenscyclus/probleemanalyse -->
-
-??? simple "Ontwerp"
-
-    <!-- list_vereisten_1 levenscyclus/ontwerp -->
-
-??? simple "Dataverkenning en datapreparatie"
-
-    <!-- list_vereisten_1 levenscyclus/dataverkenning-en-datapreparatie -->
-
-??? simple "Ontwikkelen"
-
-    <!-- list_vereisten_1 levenscyclus/ontwikkelen -->
-
-??? simple "Verificatie en validatie"
-
-    <!-- list_vereisten_1 levenscyclus/verificatie-en-validatie -->
-    
-??? simple "Implementatie"
-
-    <!-- list_vereisten_1 levenscyclus/implementatie -->
-    
-??? simple "Monitoring en beheer"
-
-    <!-- list_vereisten_1 levenscyclus/monitoring-en-beheer -->
-    
-??? simple "Uitfaseren"
-
-    <!-- list_vereisten_1 levenscyclus/uitfaseren -->
+> Impactvolle algoritmes en AI-systemen worden gepubliceerd in het Nederlandse algoritmeregister.
+
+Deze eis geldt alleen voor impactvolle algoritmes en hoog-risico-AI-systemen. Rekenregels hoef je niet te registreren in het algoritmeregister.
+
+## Overzicht eisen
```

Comment:
Dit moet wat mij betreft ook geautomatiseerd worden gemaakt. Anders te foutgevoelig, en moeten dingen op meerdere plekken worden aangepast. Wel prima om er 1 lijst van te maken. Ik denk dat dit een goede plek zou zijn voor de filterfunctionaliteit waar @rubenrouwhof mee bezig is!

Reply:
Ik zie dat je er een geautomatiseerd lijstje van hebt gemaakt, maar in mijn preview-functie werkt dit niet. Ik zie nu helemaal geen eisen meer. Misschien nog even dubbelchecken?